### PR TITLE
Fixed port issue by creating dev-override.env file

### PR DESCRIPTION
**Pull request by kharakkular**

### CODE REVIEW

File: src/configProvider.ts [hunk 1]
```diff
@@ -23,17 +24,25 @@ interface ConfigInstance {
...
   SENDGRID_PRINT_ONLY: boolean;
+  PORT: string;
 }
 let instance: ConfigInstance;
 
 export default () => {
   if (!instance) {
     const envFilePath = getConfig("MB_ENV_FILE");
-
+    const overrideEnvFilePath = getConfig("OVR_MB_ENV_FILE");
```
Comment (monarchwadia) [inline src/configProvider.ts:34]: Can we call this `MB_ENV_FILE_OVR` ?

**Review by monarchwadia — CHANGES_REQUESTED** [review]

File: src/configProvider.ts (hunk 1 above)
Comment (kharakkular) [reply src/configProvider.ts:34]: ok

File: dev/dev-overrides.env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+PORT=3000
```
Comment (monarchwadia) [inline dev/dev-overrides.env:1]: Couple of requests:
1. Can we place this file in the root of the project?
1. Can we gitignore this file?

File: src/configProvider.ts [hunk 3]
```diff
@@ -23,25 +24,36 @@ interface ConfigInstance {
...
+    const envFilePath = getConfig("MB_ENV_FILE");
+    
+    const overrideEnvFilePath = getConfig("MB_ENV_FILE_OVR");
+    
     config({
       path: path.join(__dirname, `..`, envFilePath),
     });
 
+    // override
+    const envConfig = parse(fs.readFileSync(path.join(__dirname, "..", overrideEnvFilePath)));
```
Comment (monarchwadia) [inline src/configProvider.ts:43]: what happens if the file does not exist? (this is the default scenario)

**Review by monarchwadia — CHANGES_REQUESTED** [review]
Thanks for the PR, Kharak! I have a few change requests. Let me know if you have any questions :-)

File: dev/dev-overrides.env (hunk 2 above)
Comment (kharakSinghkular) [reply dev/dev-overrides.env:1]: ok, sure

Comment (kharakkular) [reply dev/dev-overrides.env:1]: are we changing the location of all the .env files to root directory or only this file?

File: src/configProvider.ts (hunk 3 above)
Comment (kharakkular) [reply src/configProvider.ts:43]: also, if the dev-override.env file is not present then the values will be picked from dev.env and i have tested that. It works only if the file is present in the project at the specified location mentioned in the readme file

File: dev/dev-overrides.env (hunk 2 above)
Comment (kharakSinghkular) [reply dev/dev-overrides.env]: Done

File: src/configProvider.ts (hunk 3 above)
Comment (kharakSinghkular) [reply src/configProvider.ts]: done

**Review by monarchwadia — APPROVED** [review]

**Pull request merged**